Remove unused `themeChanged()` from `RenderTheme.h`

Ahmad-S792 opened the pull request:
#### 70e260f9780a291d5b73a554ca21071be835bb40
<pre>
Remove unused `themeChanged()` from `RenderTheme.h`

<a href="https://bugs.webkit.org/show_bug.cgi?id=270611">https://bugs.webkit.org/show_bug.cgi?id=270611</a>

Reviewed by Aditya Keerthi.

This patch removes unused function &apos;themeChanged()&apos; from `RenderTheme.h`.

* Source/WebCore/rendering/RenderTheme.h:

Canonical link: <a href="https://commits.webkit.org/275817@main">https://commits.webkit.org/275817@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a137331abb563eedef4da4b590566ee8b695584d

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/42898 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/21919 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/45299 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/45516 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/39023 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/45204 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/25618 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/19299 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/35503 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/43471 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/18944 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/36935 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/16475 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/16554 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/37993 "Passed tests") | [✅ 🛠 wpe-skia](https://ews-build.webkit.org/#/builders/52/builds/954 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/39075 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/38328 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/47039 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/17725 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/14616 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/42251 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/19350 "Built successfully") | | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/40904 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/9569 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/19514 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/18980 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/ec178d2f43bbf97819f72baea0b1952bdbd87a23. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/25562

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/66f45dc5766050df47bc6d0bffa1c7e7514bb165. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/25562

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/a137331abb563eedef4da4b590566ee8b695584d. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/25562

Ahmad-S792 committed:
Remove unused `themeChanged()` from `RenderTheme.h`

https://bugs.webkit.org/show_bug.cgi?id=270611

Reviewed by Aditya Keerthi.

This patch removes unused function 'themeChanged()' from `RenderTheme.h`.

* Source/WebCore/rendering/RenderTheme.h:

Canonical link: https://commits.webkit.org/275817@main

webkit-commit-queue commented:
Committed 275817@main (70e260f9780a): <https://commits.webkit.org/275817@main>

Reviewed commits have been landed. Closing PR #25562 and removing active labels.
